sdk/src/connections/usb/windows/usb_storage_windows.cpp

```diff
@@ -81,7 +81,7 @@ Status UsbStorage::read(const uint32_t address, uint8_t *data,
 
     HRESULT hr = UsbWindowsUtils::UvcExUnitReadBuffer(
         m_implData->handle->pVideoInputFilter, 5, m_implData->id, address, data,
-        bytesCount);
+        static_cast<unsigned int>(bytesCount));
```

Comment:
why 'unsigned int'?

Reply:
UvcExUnitReadBuffer accepts an uint32_t parameter.

Reply:
so why not cast to uint32_t?

Reply:
I thought they are pretty much the same.

Reply:
unsigned int is guaranteed to be at least 16 bits. So it varies based on platform. uin32_t is guaranteed to be exactly 32 bits.